src/plugin/arguments/src/commonMain/kotlin/community/flock/wirespec/plugin/CompilerArguments.kt

```diff
@@ -0,0 +1,24 @@
+package community.flock.wirespec.plugin
+
+data class CompilerArguments(
```

Comment:
This is not about compiler arguments anymore, is it? But rather about plugin arguments?